test/service/test_manifest.py

```diff
@@ -1195,6 +1195,28 @@ def test_full_metadata_missing_fields(self):
         self.assertGreater(len(intersection), 0)
         self.assertGreater(len(symmetric_diff), 0)
 
+    @manifest_test
+    def test_curl_format_manifest(self):
+        self.maxDiff = None
+        bundle_fqid = BundleFQID('f79257a7-dfc6-46d6-ae00-ba4b25313c10', '2018-09-14T133314.453337Z')
+        self._index_canned_bundle(bundle_fqid)
+        filters = {'fileFormat': {'is': ['pdf']}}
+        response = self._get_manifest(ManifestFormat.curl, filters)
+        self.assertEqual(200, response.status_code, 'Unable to download manifest')
```

Comment:
```suggestion
        self.assertEqual(200, response.status_code)
```

Documenting or explaining the obvious actually makes code harder to read due to the increased verbosity.